src/ParserRuleContext.ts

```diff
@@ -210,13 +210,21 @@ export class ParserRuleContext extends RuleContext {
 		throw new Error("The specified node does not exist");
 	}
 
+	get children(): ParseTree[] {
```

Comment:
I think there may be a better declaration for the return type:  `ReadOnlyArray<ParseTree>`.   I haven't tried this yet, but I think it may enhance the safety of returning the underlying member by generating errors on attempts to mutate it.